### PR TITLE
[DO NOT MERGE] General update of danish translations

### PR DESCRIPTION
**Pull request by Hjart**

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -606,7 +606,7 @@ shop-convenience
 en:4Convenience Store|shop|U+1F3EC|U+1F3EA
 ar:متجر|متجر صغير|المتجر
 cs:Smíšené zboží|koloniál|obchod
-da:Dagligvarebutik|døgnbutik|døgnkiosk|kiosk|butik
+da:døgnbutik|døgnkiosk|kiosk|butik
```
Comment (therearesomewhocallmetim) [inline data/categories.txt]: The first word should begin with a capital letter.

File: data/categories.txt [hunk 2]
```diff
@@ -5677,7 +5677,7 @@ en:Emergency Phone
 ru:Телефон для экстренных вызовов
 ar:هاتف الطوارئ
 cs:Tísňového volání
-da:Emergency telefon
+da:nødtelefon
```
Comment (therearesomewhocallmetim) [inline data/categories.txt]: The first letter should be capital

File: data/categories.txt [hunk 3]
```diff
@@ -6611,7 +6611,7 @@ en:Entrance
 ru:Проход
 ar:مدخل الحاجز
 cs:Vstup
-da:Indgang
+da:gennemgang
```
Comment (therearesomewhocallmetim) [inline data/categories.txt]: Should start with a capital letter

**Comment by Zverik** [comment]
Hi, could you please sign [the contributor's agreement](http://maps.me/cla/) so we can use your edits?

**Comment by therearesomewhocallmetim** [comment]
I used these translations in another pull request.

**Pull request closed without merging**